packages/openapi-ts/src/plugins/@pinia-colada/sdk/config.ts

```diff
@@ -0,0 +1,21 @@
+import type { Plugin } from '../../types';
+import { handler } from './plugin';
+import type { Config } from './types';
+
+export const defaultConfig: Plugin.Config<Config> = {
+  _dependencies: ['@hey-api/typescript'],
+  _handler: handler,
+  exportFromIndex: true,
+  name: '@pinia-colada/sdk',
+  output: 'queries',
```

Comment:
This file would then also be named `colada` if you agree with the change above. We might allow users to rename it one day, but for now I'd prefer consistency over customisation